spec/std/range_spec.cr

```diff
@@ -384,6 +384,35 @@ describe "Range" do
     end
   end
 
+  describe "sample" do
+    it "raises on open range" do
+      expect_raises(ArgumentError, "Can't sample an open range") do
+        (1..(true ? nil : 1)).sample
+      end
+      expect_raises(ArgumentError, "Can't sample an open range") do
+        ((true ? nil : 1)..1).sample
+      end
+      expect_raises(ArgumentError, "Can't sample an open range") do
+        ((true ? nil : 1)..(true ? nil : 1)).sample
+      end
+    end
+
+    it "samples a float range as a distribution" do
+      (1.2..3.4).should contain((1.2..3.4).sample)
```

Comment:
This is testing `(1.2..3.4).sample`, not `1.2..3.4`. So expected and actual value should be switched. Unfortunately there is no dedicated spec helper (#4532) and the only option is to use `includes?` or `in?` with `be_true` expectation like in the enumarble specs above. Ditto for the next example.

Reply:
This isn't resolved. Looks like your commits might have went wrong?

Reply:
@straight-shoota I don't understand what's the problem here

Reply:
An expectation should generally be of the form `actual.should matcher(expected)`. This one is `expected.should matcher(actual)`. I admit it's not a huge issue, but I would still prefer to have this fixed.
The expectations added to `enumerable_spec` do this right.

Reply:
Can you suggest a line to fix this?

Reply:
Already done.